docs/source/api/v4/oc_ci_configuration.rst

```diff
@@ -31,13 +33,13 @@ Triggers an asynchronous task to update the configuration for the :abbr:`uCDN (U
 
 Request Structure
 -----------------
-.. table:: Request Required Headers
+.. table:: Optional Request Headers
 
 	+-----------------+------------------------------------------------------------------------------------------------------------------------------+
 	|    Name         | Description                                                                                                                  |
 	+=================+==============================================================================================================================+
 	|  Authorization  | A :abbr:`JWT (JSON Web Token)` provided by the :abbr:`dCDN (Downstream Content Delivery Network)` to identify the            |
-	|                 | :abbr:`uCDN (Upstream Content Delivery Network)`                                                                             |
+	|                 | :abbr:`uCDN (Upstream Content Delivery Network)`. This can be proivded as a header or a cookie.                              |
```

Comment:
this was meant just so that it follows the SVA spec for CDNi endpoints but not have bigger impacts on our full system to minimize effort and make sure we dont break anything. but reading the spec again it says that the JWT is typically in the authorization header, not that it must be so im now debating just removing that functionality from these endpoints and leaving it as a cookie

Reply:
I think it's great to put the token in the Authorization header, I just think it's easier to support that as a general authorization method (i.e. instead of the `mojolicious` cookie) for the whole API, rather than only sending the header for specific endpoints.

Reply:
fair. ill remove it from this PR since it would be better as its own PR i think. with this JWT change we were trying to keep it as "break safe" as possible so im not sure if adding in Authorization header support would require a mailing list discussion or not. but i dont think it should be part of this PR either way